components/raftstore/src/store/fsm/store.rs

```diff
@@ -115,6 +115,14 @@ pub struct StoreInfo<EK, ER> {
     pub capacity: u64,
 }
 
+/// A trait that provide the meta information that can be accessed outside
+/// of raftstore.
+pub trait ExternStoreMeta: Send {
```

Comment:
It may not be a good name. Any suggestion is welcome.

Reply:
maybe StoreRegionMeta as the trait's function is about region related search in the store level?